fluent.ebnf

```diff
@@ -16,13 +16,15 @@ keyword              ::= [a-zA-Z_.?-] ([a-zA-Z0-9_.?- ]* [a-zA-Z0-9_.?-])?
 builtin              ::= [A-Z_.?-]+
 number               ::= [0-9]+ ('.' [0-9]+)?
 
-member-key           ::= number | (identifier '/')? keyword
-member               ::= '[' member-key ']' __ pattern NL
-default-member       ::= '*' member
-variants-list        ::= NL (__ member)* __ default-member (__ member)*
-traits-list          ::= variants-list | NL (__ member)+
+variant-key          ::= number | keyword
+variant              ::= '[' variant-key ']' __ pattern NL
+default-variant      ::= '*' variant
+variant-list         ::= NL (__ variant)* __ default-variant (__ variant)*
 
-message              ::= identifier __ '=' __ (pattern | pattern traits-list | traits-list)
+attribute            ::= '.' identifier __ '=' __ pattern NL
+attribute-list       ::= NL (__ attribute)+
+
+message              ::= identifier __ '=' __ (pattern | pattern attribute-list | attribute-list)
```

Comment:
Each `entry` ends with `NL` too, so I think we end up with an new line at the end in all scenarios.